### ISSUE TITLE
About the maze environments

### ISSUE THREAD

**Issue by yingchengyang** [Baichenjia/CeSD#2]
Thanks for such a wonderful work! I'm curious about the experiments of Maze environments as shown in Fig. 3, 7, and 8 of the paper. Is there the code for these Maze environments? Thanks a lot!

**Comment by Cho-Geonwoo** [comment]
Hi @yingchengyang,

I’m running into the same issue—I'd love to try out the Maze environments from Figs. 3, 7, and 8 but don’t see the code in the repo. Were you ever able to get access to them or find a workaround? Any pointers would be hugely appreciated! Thank you!

**Comment by yingchengyang** [comment]
Yes, you can find the maze environment in other repos, like https://github.com/victorcampos7/edl or https://github.com/Rooshy-yang/BeCL-2D-Maze.

**Comment by yingchengyang** [comment]
Hi~@Cho-Geonwoo 

Yes, you may find the maze environments in other repos, like https://github.com/victorcampos7/edl or https://github.com/Rooshy-yang/BeCL-2D-Maze

**Comment by Cho-Geonwoo** [comment]
Thanks so much for the kind pointer!! 🙏
